### PR TITLE
ngql overview

### PR DESCRIPTION
**Pull request by izhuxiaoqing**

### CODE REVIEW

File: docs-2.0/3.ngql-guide/1.nGQL-overview/1.overview.md [hunk 1]
```diff
@@ -41,101 +44,145 @@ For details of the symbols in nGQL syntax, see the following table:
...
+!!! Note
 
-    Nebula Graph Query Language does not support `SPARQL` nor `GraphQL`.
+    `nGQL` = `native nGQL` + `openCypher compatible sentences`
+
+### Is nGQL compatible with openCypher 9?
+
+!!! compatibility
+  
+  nGQL is partially compatible with openCypher 9.
```
Comment (FengUyu) [inline docs-2.0/3.ngql-guide/1.nGQL-overview/1.overview.md:80]: There is a format error. If "nGQL is partially compatible with openCypher 9" is the title, please use：
!!! compatibility "nGQL is partially compatible with openCypher 9."

File: docs-2.0/3.ngql-guide/1.nGQL-overview/1.overview.md [hunk 2]
```diff
@@ -41,101 +44,145 @@ For details of the symbols in nGQL syntax, see the following table:
...
+|Label and tag| A label is used for searching a vertex, namely an index of vertex. | A tag defines the type of a vertex and its corresponding properties. It cannot be used as an index. |
+| Pre-compiling and parameterized query  | support | not supported |
+
+!!! compatibility
+  
+    [OpenCypher](http://www.opencypher.org/) 9 and [Cypher](https://neo4j.com/developer/cypher/) have some differences in grammar and licence. For example,
+    
+    Cypher requires that **All Cypher statements are explicitly run within a transaction**. While openCypher has no such requirement. And nGQL does not support transactions.
+
+    Cypher has a variety of constraints, including Unique node property constraints, Node property existence constraints, Relationship property existence constraints, and Node key constraints. While OpenCypher has no such constraints. As a strong schema system, most of the constraints mentioned above can be solved through schema definitions (including NOT NULL) in nGQL. The only function that cannot be supported is the UNIQUE constraint.
```
Comment (FengUyu) [inline docs-2.0/3.ngql-guide/1.nGQL-overview/1.overview.md:102]: Maybe you can use an ordered list for segmentation.

File: docs-2.0/3.ngql-guide/1.nGQL-overview/1.overview.md [hunk 3]
```diff
@@ -41,101 +44,145 @@ For details of the symbols in nGQL syntax, see the following table:
...
+|`Feature`|Describes the topic of the current `.feature` file.|
+|`Background`|Describes the background information of the current `.feature` file.|
+|`Given`|Describes the prerequisites of running the test statements in the current `.feature` file.|
+|`Scenario`|Describes the scenarios. If there is the `@skip` before one `Scenario`, this scenario may not work and do not use it as a working example in a production environment.|
+|`When`|Describes the nGQL statement to be executed. It can be a `executing query` or `profiling query`.|
+|`Then`|Describes the expected return results of running the statement in the `When` clause. If the return results in your environment do not match the results described in the `.feature` file, submit an [issue](https://github.com/vesoft-inc/nebula-graph/issues) to inform the Nebula Graph team.|
+|`And`|Describes the side effects of running the statement in the `When` clause.|
+| `@skip` | This test case will be skipped. Commonly, the to-be-tested code is not ready.|
+
+Welcome to [add more practical scenarios](../15.contribution/how-to-contribute.md) and become a Nebula Graph contributor.
```
Comment (FengUyu) [inline docs-2.0/3.ngql-guide/1.nGQL-overview/1.overview.md:176]: This link is wrong

File: docs-2.0/3.ngql-guide/1.nGQL-overview/1.overview.md [hunk 4]
```diff
@@ -1,21 +1,21 @@
...
 - Supports pattern match
 - Supports aggregation
 - Supports graph mutation
 - Supports access control
 - Supports composite queries
 - Supports index
-- Supports most openCypher 9 graph query syntax (but mutations and controls syntax are not supported).
+- Supports most openCypher 9 graph query syntax (but mutations and controls syntax are not supported)
 
 ## Example Data
```
Comment (FengUyu) [inline docs-2.0/3.ngql-guide/1.nGQL-overview/1.overview.md:21]: Here is a link to a GitHub page, which should give users a download link.

**Review by randomJoe211 — APPROVED** [review]

**Review by FengUyu — APPROVED** [review]

**Pull request merged**